### PR TITLE
APIS-3373 - combine pages into single application page

### PR DESCRIPTION
**Pull request by pbromley**

### CODE REVIEW

File: app/connectors/ApplicationConnector.scala [hunk 1]
```diff
@@ -90,7 +91,9 @@ trait ApplicationConnector {
   def fetchApplicationsByEmail(email:String)(implicit hc: HeaderCarrier): Future[Seq[Application]] = {
     http.GET[Seq[ApplicationResponse]](s"$applicationBaseUrl/developer/applications", Seq("emailAddress" -> email))
       .recover {
-        case e => throw new FetchApplicationsFailed
+        case e =>
+          Logger.error("ROSE", e)
```
Comment (edjannoo) [inline app/connectors/ApplicationConnector.scala]: Remove this line 😆

Comment (pbromley) [reply app/connectors/ApplicationConnector.scala]: done

Comment (peteslater-ee) [inline app/connectors/ApplicationConnector.scala]: Really?

File: test/unit/views/helper/application/ApplicationReviewSpec.scala [hunk 2]
```diff
@@ -0,0 +1,77 @@
...
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package unit.views.helper.application
+
+
+import java.time.format.DateTimeFormatter
```
Comment (peteslater-ee) [inline test/unit/views/helper/application/ApplicationReviewSpec.scala]: Seems to be unused

File: app/connectors/ApplicationConnector.scala (hunk 1 above)
Comment (pbromley) [reply app/connectors/ApplicationConnector.scala]: yeah thats gone

File: test/unit/views/helper/application/ApplicationReviewSpec.scala (hunk 2 above)
Comment (pbromley) [reply test/unit/views/helper/application/ApplicationReviewSpec.scala]: intellij importing the wrong one - fixed now

File: test/unit/utils/ApplicationGenerator.scala [hunk 3]
```diff
@@ -0,0 +1,51 @@
...
+      approvedOn = dateTime,
+      verified = true
+    )
+
+  def aCheckInformation(): CheckInformation =
+    new CheckInformation(contactDetails = Some(new ContactDetails("contactFullName", "contactEmail", "contactTelephone")),
+      confirmedName = true, providedPrivacyPolicyURL = true, providedTermsAndConditionsURL = true,
+      applicationDetails = Some("application details"))
+
+}
```
Comment (peteslater-ee) [inline test/unit/utils/ApplicationGenerator.scala]: This whole file could have the `new` label removed from the various case class creations.

Comment (pbromley) [reply test/unit/utils/ApplicationGenerator.scala]: ah of course, thats the java dev in me. 

Done

**Pull request merged**